inference-engine/samples/benchmark_app/inputs_filling.cpp

```diff
@@ -310,6 +310,8 @@ void fillBlobs(const std::vector<std::string>& inputFiles,
                 fillBlobRandom<short>(inputBlob);
             } else if (item.second->getPrecision() == InferenceEngine::Precision::I32) {
                 fillBlobRandom<int32_t>(inputBlob);
+            } else if (item.second->getPrecision() == InferenceEngine::Precision::I64) {
+                fillBlobRandom<int64_t>(inputBlob);
```

Comment:
What about filling inputs with binary input file?

Also please check python benchmark app for appropriate changes.